src/encoded/static/components/search.js

```diff
@@ -530,8 +530,8 @@ globals.listingViews.register(Target, 'Target');
 class PatientComponent extends React.Component {
     render() {
         const result = this.props.context;
-        const age = (result.age && result.age !== 'unknown') ? ` ${result.age}` : '';
-        const ageUnits = (result.age_units && result.age_units !== 'unknown' && age) ? ` ${result.age_units}` : '';
+        const age = (result.diagnosis.age && result.diagnosis.age !== 'Unknown') ? ` ${result.diagnosis.age}` : '';
+        const ageUnits = (result.diagnosis.age_unit && result.diagnosis.age_unit !== 'Unknown' && age) ? ` ${result.diagnosis.age_unit}` : '';
 
         return (
```

Comment:
![image](https://user-images.githubusercontent.com/38848214/83534705-cd161100-a4b6-11ea-9d90-001f33cf909a.png)

Reply:
Add unknown to visualization

Reply:
I actually hide the brackets when age is unknown. I feel that unknown may cause confusion.

Reply:
oh i like that member